packages/cli-repl/src/cli-repl.ts

```diff
@@ -259,8 +260,7 @@ class CliRepl {
     const version = this.buildInfo.version;
 
     this.repl = repl.start({
-      prompt: `$ mongosh > `,
-      ignoreUndefined: true,
+      prompt: `msh > `,
```

Comment:
Is even just `>` possible?

Reply:
There as a plan to include topology information in the prompt, which would be easy to implement but we haven't gotten around to it. For example, on the old shell indicates if it's a replset/etc and what the version of the server is, like `MongoDB Enterprise >`. We should make a ticket to track that and save it for post-beta?

Reply:
i will strongly prefer `>` unless we display replica set name. So if @mmarcon agrees, I will gladly just change it to `>`

Reply:
Sounds good to me. `mongo` has the same.